kfs-core/src/main/java/edu/arizona/kfs/fp/document/ProcurementCardDocument.java

```diff
@@ -0,0 +1,208 @@
+package edu.arizona.kfs.fp.document;
+
+import java.sql.Date;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.List;
+
+import org.kuali.kfs.fp.batch.ProcurementCardLoadStep;
+import org.kuali.kfs.sys.businessobject.GeneralLedgerPendingEntry;
+import org.kuali.kfs.sys.businessobject.GeneralLedgerPendingEntrySourceDetail;
+import org.kuali.kfs.sys.businessobject.SourceAccountingLine;
+import org.kuali.kfs.sys.businessobject.TargetAccountingLine;
+import org.kuali.kfs.sys.businessobject.GeneralLedgerPendingEntrySequenceHelper;
+import org.kuali.kfs.sys.context.SpringContext;
+import org.kuali.rice.kim.api.group.GroupService;
+import org.kuali.rice.krad.util.ObjectUtils;
+import org.kuali.kfs.sys.service.UniversityDateService;
+import org.kuali.rice.coreservice.framework.parameter.ParameterService;
+
+import edu.arizona.kfs.fp.businessobject.ProcurementCardHolder;
+import org.apache.commons.lang.StringUtils;
+
+import edu.arizona.kfs.fp.businessobject.ProcurementCardTransactionDetail;
+import edu.arizona.kfs.fp.service.ProcurementCardService;
+
+public class ProcurementCardDocument extends org.kuali.kfs.fp.document.ProcurementCardDocument {
+    private static final org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(ProcurementCardDocument.class);
+    
+    /**
+     * Default serial version UID, courtesy of Eclipse
+     */
+    private static final long serialVersionUID = 1L;
```

Comment:
This can actually be a problem, we should generate a real long for this purpose. (It plays into how things get serialized/de-serialized).